### DISCUSSION TITLE
Flask  Issues, Any idea ?

### DISCUSSION THREAD

**Discussion by t-dmac** [kyleskom/NBA-Machine-Learning-Sports-Betting#461] [General]
I'm having this issue when running the flask. 

Any idea what's going on ? 

![Screenshot 2024-12-19 at 4 46 41 PM](https://github.com/user-attachments/assets/a388966f-4b32-4db5-9b5f-b936b0fc5b92)

**Comment by kyleskom** [comment]
I have not seen this one, id suggest asking gpt.
